core/runner.py

```diff
@@ -284,12 +281,11 @@ def _transform_and_split_data(self, job):
         session_filepath = os.path.join(transformed_filepath, new_name)
         os.makedirs(session_filepath)
 
-        # 4. Retrieve transform function and train-test split from job.
-        transform_function = job.transform_function
+        # 4. Retrieve train-test split from job.
         split = job.hyperparams['split']
 
         # 5. Shuffle raw data, then split into train and test set.
-        transformed_data = transform_function(raw_data)
+        transformed_data = raw_data
```

Comment:
Since there's no more transform function, you should just delete this line and s/transformed_data/raw_data since it would be misleading otherwise.

Reply:
Done